### PR TITLE
Right Click edits.

### PR DESCRIPTION
**Pull request by shadrackkibet**
@dannyparsons this are minor edits suggested at #3979

### CODE REVIEW

File: instat/ucrDataView.vb [hunk 1]
```diff
@@ -47,6 +47,8 @@ Public Class ucrDataView
         mnuFreezeToHere.Enabled = False
         mnuUnfreeze.Enabled = False
         grdData.Visible = False
+        mnuInsertColsBefore.Enabled = False
+        mnuInsertColsAfter.Enabled = False
```
Comment (dannyparsons) [inline instat/ucrDataView.vb]: Let's hide them instead of disabling. And remember any change here you must also make on the column metadata right click.

Comment (shadrackkibet) [reply instat/ucrDataView.vb]: We also have freeze disabled.Shall i hide this as well?

Comment (dannyparsons) [reply instat/ucrDataView.vb]: I think freeze we probably want to get working eventually so good to leave it there so people see it is coming.

Comment (shadrackkibet) [reply instat/ucrDataView.vb]: great!

Comment (shadrackkibet) [reply instat/ucrDataView.vb]: @dannyparsons i have now changed this.

**Comment by dannyparsons** [comment]
Great, is levels/labels spelt correctly on column metadata already?

**Comment by shadrackkibet** [comment]
yep, that is spelt correctly.

**Pull request merged**